packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/ContentEdit.ts

```diff
@@ -43,6 +45,11 @@ export default class ContentEdit implements EditorPlugin {
      * @param editor The editor instance
      */
     initialize(editor: IEditor): void {
+        this.editor = editor;
+        this.addFeatures(editor);
```

Comment:
suggest cache the feature array in this plugin class. Then when dispose, just remove them one by one from this array, no need to get them again.